File: Java/Persona.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package exoneracion;

/**
 *
 * @author juanp
 */
public abstract class  Persona {
    public String Nombre;
    public int Edad;
    private String cedula;
    
    public Persona(String Nombre, int edad, String cedula) {
        this.Nombre = Nombre;
        this.Edad = edad;
    }

    public void setCedula(String nuevaCedula) {
        this.cedula = nuevaCedula;
    }
    
    public abstract void comer();
    public abstract void Dormir();

}

File: Primer Parcial/C++/Persona.cpp
#include <iostream>
#include <string>

class Persona {
public:
    std::string Nombre;
    int Edad;

private:
    std::string cedula;

public:
    Persona(std::string Nombre, int edad, std::string cedula)
        : Nombre(Nombre), Edad(edad), cedula(cedula) {}

    void setCedula(std::string nuevaCedula) {
        this->cedula = nuevaCedula;
    }

    virtual void comer() = 0;
    virtual void Dormir() = 0;
};
